src/Lime.Protocol/LimeUri.cs

```diff
@@ -140,5 +150,17 @@ public static Uri GetBaseUri(Identity authority)
         public static implicit operator LimeUri(string value) => value == null ? null : Parse(value);
         
         public static implicit operator string(LimeUri limeUri) => limeUri?.ToString();
+
+        // TODO: Remove this method once the 'if' statement on the ctor is removed
+        private bool ReceivedUriPathIsEncoded(string uri)
+            => WebUtility.UrlDecode(uri) != uri;
+
+        private void ValidateAbsoluteUri(Uri absoluteUri)
```

Comment:
Nice, but don't you think that this method have a more specific name, like `ValidateLimeScheme` or something?